apps/dashboard/src/app/ApplicationShell.tsx

```diff
@@ -33,7 +33,7 @@ const navigations = [
     icon: "tabler:moneybag",
     children: [
       { label: "Bedrifter", href: "/company" },
-      { label: "Utlysninger", href: "/listing" },
+      { label: "Utlysninger", href: "/joblisting" },
```

Comment:
The reason I went for this approach rather than using kebab-case was that in `packages/core` it avoids confusion with regards to it just being one table, its not joining anything like for example event-company etc.  But I guess job-listing or jobListing are the idiomatic approaches. I can go with job-listing if you want, I'm not very opinionated here.